.github/workflows/mkdocs-publish.yml

```
@@ -0,0 +1,41 @@
+name: publish mkdocs
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  publish-mkdocs:
+    runs-on:
+      - ubuntu-latest
```

make it consistent with `pre-commit` workflow
```suggestion
      - ubuntu-22.04
```

---

Let's do it the other way around then.

---

sure